docs/source/commandline/commandline-all.rst

```diff
@@ -1526,63 +2118,92 @@ Debugging Options
 
 **-[no]verbose** / **-[no]v**
 
-	Displays internal diagnostic information. This information is very useful for debugging problems with your configuration. IMPORTANT: when reporting bugs, please run with **mame -verbose** and include the resulting information. The default is OFF (**-noverbose**).
+    Displays internal diagnostic information. This information is very useful
+    for debugging problems with your configuration.  IMPORTANT: when reporting
+    bugs, please run with **mame -verbose** and include the resulting
+    information.
+
+    The default is OFF (**-noverbose**).
 
 .. _mame-commandline-oslog:
 
 **-[no]oslog**
 
-	Output error.log data to the system debugger. The default is OFF (**-nooslog**).
+    Output error.log data to the system debugger.
```

Comment:
This is OS-dependent.  On Windows it sends it to the debugger if one is attached (otherwise it goes nowhere) while on other operating systems it goes to standard error output and typically ends up in the terminal window or on the console.

Reply:
Done.